Sources/Danger/Extensions/DateFormatterExtensions.swift

```diff
@@ -6,6 +6,23 @@ extension DateFormatter {
         let dateFormatter = OptionalFractionalSecondsDateFormatter()
         return dateFormatter
     }
+
+    public static var onlyDateDateFormatter: DateFormatter {
+        let dateFormatter = OptionalFractionalSecondsDateFormatter.onlyDate
+        return dateFormatter
+    }
+
+    /// Handles multiple date format inside models.
+    public static func dateFormatterHandler(_ decoder: Decoder) throws -> Date {
+        let dateString = try decoder.singleValueContainer().decode(String.self)
+        if let date = defaultDateFormatter.date(from: dateString) {
+            return date
+        } else if let date = onlyDateDateFormatter.date(from: dateString) {
+            return date
+        } else {
+            fatalError("Unexpected date coding key: \(decoder.codingPath.last?.stringValue ?? "Not Valid Key Name")")
```

Comment:
Not sure are there any better ways to handle this kind of situation.

Reply:
[imo]
it would be nice that define a new `Error` like

```swift
extension OptionalFractionalSecondsDateFormatter {
    enum Errors: Error {
        case invalidFormat(String)
    }
}
```

and throw instead of `fatalError`.

Reply:
Thanks for the advice!